### DISCUSSION TITLE
Source code editor and code style

### DISCUSSION THREAD

**Discussion by XspeedPL** [ErinaSugino/Starbound-SBR-Mod#67] [General]
What kind of source code editor are you using for the project?
Ideally contributors should use the same one to keep code style consistent.

**Comment by ErinaSugino** [comment]
I'm using Notepad++. Doesn't necessarily classify as a code editor though, just a fancily styled text editor, but it's easy to use and does the job (minus potential syntax error checking, but I don't know how well something like VS would fare with Lua too).

**Reply by XspeedPL** [reply]
Personally I am using VS Code which is free and handles Lua very well. It requires some minor setup to recognize Starbound globals, but otherwise even without that it can help getting rid of basic syntax errors and does static null checking

**Reply by ErinaSugino** [reply]
Guess I could take a look into that for the future then. Because every now and then some refactoring support and reference lookup would be nice.
